lib/pure/unittest.nim

```diff
@@ -515,7 +515,9 @@ template suite*(name, body) {.dirty.} =
     finally:
       suiteEnded()
 
-template exceptionTypeName(e: typed): string = $e.name
+proc exceptionTypeName(e: ref Exception): string {.inline.} =
+  if e == nil: "<foreign exception>"
```

Comment:
I like the change, 
but is possible to detect that the target is JavaScript then use:
`"<JavaScript exception>"` ?.  :thinking:

Reply:
@juancarlospaco I think we can do even better, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name 
do you want to take a stab at it?